doc/manpages/vtysh.rst

```diff
@@ -41,6 +41,20 @@ OPTIONS available for the vtysh command:
 
    When the -c option is being used, this flag will cause the standard vtysh prompt and command to be echoed prior to displaying the results. This is particularly useful to separate the results when executing multiple commands.
 
+.. option:: -C, --dryrun
+
+   When the -C option is being used, this flag will check the config for syntatic validity.
+
+.. option:: -m, --markfile
+
+   Mark the input file with context ends, useful for cleanup of a config file
+   that has a lot of extraneous space and end markers
+
+.. option:: -n, --noerror
+
+   When executing cli that does not invoke a vtysh shell, if an error ocurrs
+   ignore it for purposes of return codes from vtysh.
```

Comment:
Sorry, I forgot to mention this in the last review. If you wrap the lines yourself here, then they'll be wrapped in the manpage and `man` won't be able to properly wrap them to the terminal size. So don't break paragraphs, just put them all on one line.

Reply:
fixed